extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpHostConfigSource.java

```diff
@@ -45,19 +45,11 @@ public String getValue(String propertyName) {
             // In dev-mode we want to only listen on localhost so others on the network cannot connect to the application.
             // However, in WSL this would result in the application not being accessible,
             // so in that case, we launch it on all interfaces.
-            return (LaunchMode.current().isDevOrTest() && !isWSL()) ? "localhost" : ALL_INTERFACES;
+            return (LaunchMode.current().isDevOrTest() && !LaunchMode.isWSL()) ? "localhost" : ALL_INTERFACES;
```

Comment:
I might be blind but I don't see this method being moved to `LaunchMode`? And TBH, I'm not sure it's the right place for that.

Reply:
Ugh, you're not blind! Fixed with [e54b699](https://github.com/quarkusio/quarkus/pull/40497/commits/e54b69979168e8bd3c196cdbf325efebd9e0415d).

The LaunchMode class made sense to me because it's used both by the HttpHostConfigSource and the VertxHttpRecorder. But It's my first contribution (apart from documentation) so you'll will certainly know better places for it (as you have already suggested above).